.gitignore

```diff
@@ -202,3 +202,8 @@ docsrc/tools/FSharp.Formatting.svclog
 # Node install directory
 
 tests/FSharpPlusFable.Tests/node_modules/
+
+
+# Fable compilation files
+
+tests/FSharpPlusFable.Tests/FSharpPlusFable
```

Comment:
shouldn't this be more specific to exclude only js files? You could add a .gitignore to that folder to exclude the compilation output.

Reply:
The folder FSharpPlusFable is created when you compile F#+ with Fable through the target I've added. I've tagged you at this line. This makes git ignore any result of compiling F#+ locally.